test/server/cards/01_SOR/units/Snowspeeder.spec.js

```diff
@@ -0,0 +1,41 @@
+describe('Snowspeeder', function() {
+    integration(function () {
+        describe('Snowspeeder\'s ability -', function() {
+            beforeEach(function() {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['snowspeeder'],
+                        groundArena: ['wampa'],
+                        spaceArena: ['tieln-fighter'],
+                    },
+                    player2: {
+                        deck: ['occupier-siege-tank'],
+                        groundArena: ['cell-block-guard', 'atst']
+                    }
+                });
+            });
+
+            it('should exhaust an enemy Vehicle ground unit', function() {
+                this.player1.clickCard(this.snowspeeder);
+                this.player1.clickPrompt('Ambush');
+
+                expect(this.atst.exhausted).toBe(true);
+            });
+
+            describe('when multiple valid targets,', function() {
```

Comment:
We try to not overdo it on tests to keep our github actions usage low (the tests are extremely heavy for some reason, not sure if it's jasmine or the test setup). We typically don't bother testing both the single-target and multi-target case, do you mind removing one.

If you choose to keep the single-target case, please add a comment explaining that it is automatically targeting due to sentinel